src/app/participant-list/participant-list.component.css

```diff
@@ -1,7 +1,7 @@
 .LIST-SEPARATOR:not(:last-child) {
   border-bottom:1px solid lightgrey;
-  margin-top: 0px;
-  margin-bottom: 0px;
+  margin-top: 0;
```

Comment:
should the `px` be removed with angular10 now? it is still in src/styles.css

Reply:
It is written in spec https://www.w3.org/TR/css-values-3/.
`5. Distance Units: the <length> type
For zero lengths the unit identifier is optional (i.e. can be syntactically represented as the <number> 0).
`

I mean `px` is not necessary, we can set just `0`.
(It is not required to remove the `px`, it is just shorter)